src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+
+const updateButtons = () => {
+  const allRows = table.rows.length;
+  const allCols = table.rows[0].cells.length;
+
+  appendRow.disabled = allRows >= 10;
+  removeRow.disabled = allRows <= 2;
```

Comment:
Performance issue: instead of querying the DOM every time you need the number of rows or columns, you could store these values in variables and update them only when a row or column is added or removed.